app/src/main/kotlin/com/tailoredapps/androidapptemplate/detail/DetailView.kt

```diff
@@ -37,16 +36,16 @@ class DetailView : Fragment(R.layout.fragment_detail) {
     private val navController by lazy(::findNavController)
     private val viewModel by viewModel<DetailViewModel>()
 
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(view, savedInstanceState)
+    init {
```

Comment:
Are there any repercussions on doing this on init? or should this be done in `onCreate`?

Reply:
Hm, good question (somewhat missed this on the first run).

Just thought about the viewBinding reference being outdated when the Activity / fragment is recreated, until I saw the implementation of the `viewBinding` delegate :sweat_smile: :+1: 

Other than that, it could be an easy trap for developers to use view-related classes here (i.e. Managers :trollface:), which do not honor the lifecycle themselves (contrary to what is done in the viewBinding delegate), which would then lead to misbehavior when referencing i.e. old view elements after view recreation (which may not be identified easily, as it only occurs after recreating the view).
In regards to this I would probably like an approach better where this is moved inside the onCreate method.

Other than that, another PoV may be, that things that are invoked during onCreate instead of kotlins `init` may things, althoigh this is only a gut feeling and I can't think of any example right now.

Reply:
I think it might clash when using navigation arguments, as these are iirc not available until the `onCreate` or some other lifecycle method xD (event when using the `navArgs` extension Functions)

Reply:
But consider that the `block` of the extension is only called after the `Fragment.View.OnStart`, so that stuff should already be initialized.

Reply:
Ah true, then I have no more concerns